src/lv_core/lv_obj.h

```diff
@@ -147,6 +148,7 @@ enum {
 typedef uint8_t lv_signal_t;
 
 typedef lv_res_t (*lv_signal_cb_t)(struct _lv_obj_t * obj, lv_signal_t sign, void * param);
+typedef bool (*lv_hittest_cb_t)(struct _lv_obj_t * obj, const lv_point_t * point);
```

Comment:
Is this still required?

Reply:
Nope. I forgot to remove it. :slightly_smiling_face: